pkg/providers/tinkerbell/tinkerbell_test.go

```diff
@@ -24,6 +24,16 @@ const (
 	expectedTinkerbellPBnJGRPCAuthority = "1.2.3.4:42000"
 )
 
+type DummyProviderTinkClient struct{}
```

Comment:
nit: why build your own dummy if you are already generating the mocks for this interface?